src/web/public/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Telescope",
+  "short_name": "Telescope",
+  "theme_color": "#121D59",
+  "background_color": "#FFFFFF",
+  "display": "fullscreen",
+  "orientation": "portrait",
+  "scope": "/",
+  "start_url": "/",
+  "icons": [
+    {
+      "src": "logoImages/manifest-icon-192.png",
```

Comment:
We have a lot more icon sizes than just the 192x192 one, why only this one?

Reply:
I can add a few more. I include one for simplicity initally

Reply:
That would be good, yes.  Let's do it either in this PR, or file an issue and we can follow-up with it later.